circuit/src/utils.rs

```diff
@@ -291,3 +292,46 @@ where
 
 	Ok(res)
 }
+
+/// The u64 integer represented by an L-bit little-endian bitstring.
+///
+/// # Panics
+///
+/// Panics if the bitstring is longer than 64 bits.
+pub fn lebs2ip<const L: usize>(bits: &[bool; L]) -> u64 {
+	assert!(L <= 64);
+	bits.iter().enumerate().fold(0u64, |acc, (i, b)| acc + if *b { 1 << i } else { 0 })
+}
+
+/// Get the modulus of field as [`BigUint`].
+pub fn modulus<F: FieldExt>() -> BigUint {
+	BigUint::from_str_radix(&F::MODULUS[2..], 16).unwrap()
+}
+
+/// Get the field element of `2 ^ n`
+pub fn power_of_two<F: FieldExt>(n: usize) -> F {
+	big_to_fe(BigUint::one() << n)
+}
+
+/// Convert the [`BigUint`] value to [`Field`] element.
+pub fn big_to_fe<F: FieldExt>(e: BigUint) -> F {
+	let modulus = modulus::<F>();
+	let e = e % modulus;
+	F::from_str_vartime(&e.to_str_radix(10)[..]).unwrap()
+}
+
+/// Conver the [`Field`] element to [`BigUint`] value
+pub fn fe_to_big<F: FieldExt>(fe: F) -> BigUint {
+	BigUint::from_bytes_le(fe.to_repr().as_ref())
+}
```

Comment:
These are the copies from `integer/rns.rs`. I think this is the better place for them, so delete the ones from rns

Reply:
Yes, I done it here. https://github.com/eigen-trust/protocol/pull/169/commits/ab695818a4f077629984c7e35e847639235712ca